### PR TITLE
Invoices strings

### PR DESCRIPTION
**Pull request by FnrDev**

### CODE REVIEW

File: en/website.json [hunk 1]
```diff
@@ -658,5 +658,11 @@
     "upgrade": "Upgrade",
     "role_search": "SEARCH BY ROLE ID OR NAME",
     "type_here": "Type here ..",
-    "new_subscription": "New Subscription"
+    "new_subscription": "New Subscription",
+    "invoices_title": "Invoices Due {0}",
```
Comment (Dramex) [inline en/website.json:662]: should be `{0} Invoices Due`

File: en/website.json [hunk 2]
```diff
@@ -658,5 +658,11 @@
...
+    "invoices_description": "You have {0} Invoices(s) currently unpaid\nwith a total balance of ${1} USD",
+    "add_funds": "Add Funds",
+    "pay_all": "Pay All",
+    "premium_available_credit": "Available Credit $",
```
Comment (Dramex) [inline en/website.json:666]: `$ Available Credit`

File: en/website.json (hunk 1 above)
Comment (FnrDev) [reply en/website.json]: done

File: en/website.json (hunk 2 above)
Comment (FnrDev) [reply en/website.json]: done

**Comment by FnrDev** [comment]
> keys should be subscriptions, invoices rather than premium

fixed 7262ef834a89f40df0adac3e6713006c24d83126

**Review by Dramex — APPROVED** [review]
LGTM

**Pull request merged**